StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1121CodeFixProvider.cs

```diff
@@ -54,30 +54,39 @@ public override FixAllProvider GetFixAllProvider()
         /// <inheritdoc/>
         public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
+            var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            var semanticModel = await context.Document.GetSemanticModelAsync();
```

Comment:
The semantic model is used in line 70(old)/74(new) to check if a CF should be provided or not. So I couldn't move it there.